mwdb/resources/relations.py

```diff
@@ -99,6 +99,8 @@ def put(self, type, parent, child):
                 description: |
                     When one of objects doesn't exist or user
                     doesn't have access to object.
+            409:
+                description: Relation already exists.
```

Comment:
Hm, I'm not really convinced that we should raise an error if relation already exists (in case of PUT) or not (in case of DELETE).
Regardless of action was actually done or not: user has desired effect, which is relation added between two objects. Additional handling of these HTTP errors (e.g. due to concurrent requests) may result in unexpected problems and information that action was done exactly by our client is not so useful.